src/service/resources-impl.js

```diff
@@ -281,19 +284,61 @@ export class ResourcesImpl {
 
   /** @override */
   isIntersectionExperimentOn() {
-    return !!this.intersectionObserver_;
+    return !!this.oneViewportObserver_;
   }
 
   /**
    * @param {!Array<!IntersectionObserverEntry>} entries
-   * @param {!IntersectionObserver} unusedObserver
    * @private
    */
-  intersects_(entries, unusedObserver) {
-    dev().fine(TAG_, 'intersect', entries);
+  intersectsOneViewport_(entries) {
+    // TODO(willchou): Remove assert once #27167 is fixed.
+    devAssert(this.prerenderSize_ == 1);
 
-    const toUnload = [];
+    const whenBuilt = this.intersects_(entries, /* viewports */ 1);
 
+    // To prioritize in-viewport elements, delay handling of 2vp/3vp until
+    // all 1vp elements are built (and we're not in prerender, of course).
+    Promise.all([whenBuilt, this.ampdoc.whenFirstVisible()]).then(() => {
+      if (this.threeViewportsObserver_) {
+        return;
+      }
+      this.threeViewportsObserver_ = new IntersectionObserver(
+        e => this.intersectsThreeViewports_(e),
+        {
+          root: this.oneViewportObserver_.root,
+          rootMargin: '200% 25%',
+        }
+      );
+      const elements = this.resources_.map(r => r.element);
```

Comment:
Unsure if this is desirable, but would it be technically possible to have only a single 3p IO, and use the data in entries to calculate whether or not something intersects at viewport=1?

e.g.
```js
// takes in an entry from an IntersectionObserver with rootMargin 200% and determines if the item also intersects at 0%.
intersectsOneViewport(entryFrom3vpIO) {
  const {rootBounds, boundingClientRect, isIntersecting} = entryFrom3vpIO;
  if (!isIntersecting) {
    return false;
  }
  const vp1YTop= rootBounds.y + rootBounds.height/3;
  const vp1YBottom= rootBounds.y + rootBounds.height/3;
  const isWithinViewport = boundingClientRect.y >= vp1YTop && boundingClientRect.y <= vp1YBottom;

  // we don't care about X, since if it wasn't within our desired x-margin it would have early returned with isIntersecting=false. 
  return isWithintViewportY;
}
```

Reply:
Yea I think it's possible but less desirable. :)